tests/data/2022/NCL1/quality_control.csv

```diff
@@ -0,0 +1 @@
+﻿2022-07-28T12:30:00,2022-07-28T16:40:00

```

Comment:
```suggestion
starttime,endtime,notes
﻿2022-07-28T12:30:00,2022-07-28T16:40:00,

```

Hey @madhavmk seems like if I add a header like the above suggestion, the program breaks.